### PR TITLE
Uses old Sol Police sprites for a Police Cap reskin.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title.

## How This Contributes To The Skyrat Roleplay Experience

More flavour is always good. Plus, these look better than, what I assume is a repurposed lustvendor sprite. 

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl: Cursor.
add: Old Sol Police hat sprites are usable via Police Cap reskinning.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
These sprites **really** need redoing before they're re-added. The mob sprite looks very old, and the obj sprites even moreso.

And... the currently used hat is a bluesec recolor I made of TG's HoS Hat sprite. Not a LustWish sprite :/ 
It being used by not-the-HoS-hat is a separate issue entirely though...

**Comment by theselfish** [comment]
> These sprites **really** need redoing before they're re-added. The mob sprite looks very old, and the obj sprites even moreso.
> 
> And... the currently used hat is a bluesec recolor I made of TG's HoS Hat sprite. Not a LustWish sprite :/ It being used by not-the-HoS-hat is a separate issue entirely though...

I beg to differ. I think they look fine as is.
And, it fits the blue colour scheme more than the SS drop out hat currently in the vendor.

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm [hunk 1]
```diff
@@ -50,6 +50,13 @@
 	icon_state = "policeofficerpatrolcap"
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
 	armor_type = /datum/armor/head_helmet
+	unique_reskin = list(
+						"Blue" = "policeofficercap",
+						"Sillitoe" = "policetrafficcap",
+						"German" = "policeofficerpatrolcap",
+						"Cadet" = "policecadetcap",
+						)
```
Comment (vinylspiders) [inline modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm]:
```suggestion
		"Blue" = "policeofficercap",
		"Sillitoe" = "policetrafficcap",
		"German" = "policeofficerpatrolcap",
		"Cadet" = "policecadetcap",
	)
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]
@GoldenAlpharex for the sprites themselves

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm [hunk 2]
```diff
@@ -50,6 +50,13 @@
 	icon_state = "policeofficerpatrolcap"
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
 	armor_type = /datum/armor/head_helmet
+	unique_reskin = list(
+		"Blue" = "policeofficercap",
+		"Sillitoe" = "policetrafficcap",
+		"German" = "policeofficerpatrolcap",
+		"Cadet" = "policecadetcap",
+	)
+
```
Comment (vinylspiders) [inline modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm:59]:
```suggestion
```

**Review by vinylspiders — DISMISSED** [review]

**Comment by theselfish** [comment]
Waiting for the worms, here...

**Comment by theselfish** [comment]
@vinylspiders @GoldenAlpharex Any word what's happening here?

**Comment by theselfish** [comment]
@OrionTheFox I've fiddled around with the sprite.
image
How does this look?
Is it new enough?

**Comment by vinylspiders** [comment]
I don't see any sprite changes in the files of this PR. (that said @GoldenAlpharex will look at them once they're up). I don't think the sprites look bad though, except maybe that the choice of blue is a little faded/muted and can be even louder if you want it to really pop. But that's more of a personal thing.

**Comment by theselfish** [comment]
> I don't see any sprite changes in the files of this PR. (that said @GoldenAlpharex will look at them once they're up). I don't think the sprites look bad though, except maybe that the choice of blue is a little faded/muted and can be even louder if you want it to really pop. But that's more of a personal thing.

There aren't, yet. I was asking before I committed.

**Comment by vinylspiders** [comment]
Also, the 2nd version does look less-faded and better to me.

**Comment by vinylspiders** [comment]
> > I don't see any sprite changes in the files of this PR. (that said @GoldenAlpharex will look at them once they're up). I don't think the sprites look bad though, except maybe that the choice of blue is a little faded/muted and can be even louder if you want it to really pop. But that's more of a personal thing.
> 
> There aren't, yet. I was asking before I committed.

Go ahead and commit so we can review them properly.

**Comment by theselfish** [comment]
Sure, sure. One moment please.

**Comment by theselfish** [comment]
Committed.

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm [hunk 3]
```diff
@@ -50,6 +50,12 @@
 	icon_state = "policeofficerpatrolcap"
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
 	armor_type = /datum/armor/head_helmet
+	unique_reskin = list(
+		"Blue" = "policeofficercap",
+		"Sillitoe" = "policetrafficcap",
+		"German" = "policeofficerpatrolcap",
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm]: Do we _have_ to label it as German?

**Review by GoldenAlpharex — DISMISSED** [review]
Sprites look okay otherwise, I suppose.

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm (hunk 3 above)
Comment (theselfish) [reply modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm]: It was that or "Wanna-Be SS", if you have a better name, feel free to suggest it.

Comment (theselfish) [reply modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm]: Eh screw it, not a hill worth dying on.

Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm]:
```suggestion
		"Black" = "policeofficerpatrolcap",
```

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**